packages/venia-concept/src/components/SearchBar/searchBar.js

```diff
@@ -0,0 +1,153 @@
+import querystring from 'querystring';
```

Comment:
Is this a new dependency? I don't see it in our `package.json`.

Reply:
Thanks, I meant to flag this - it's actually [built in to Node](https://nodejs.org/api/querystring.html), but [modern browsers](https://caniuse.com/#search=URLSearchParams) include a much easier way to do it in the browser using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get).